### DISCUSSION TITLE
어떤 다국어 지원이 되는지 궁금합니다.

### DISCUSSION THREAD

**Discussion by Kwansik-Jung** [orioncactus/pretendard#130] [Q&A] [answered]
공식 홈페이지의 지원언어목록에는 다양하게 쓰여있는데
실제로 한국어, 영어, 일어 외에도 말레이시아어, 태국어, 중국어, 대만어 등이 전부 다 지원되는지 궁금합니다.

**Comment by orioncactus** [comment, marked as answer]
안녕하세요! 이제야 답변드립니다.

Pretendard는 로마자, 그리스 문자, 키릴 문자, 일본 가나, 한글을 지원합니다. 이중에서 Pretendard JP는 일본 한자 및 한국 한자를 추가적으로 지원합니다. 

말레이시아어 같은 경우에는 공식 문자로 로마자를 쓰는 것이 맞다면 무리 없이 지원 언어로 있는 것이 맞습니다. 다만 태국어는 타이 문자를 쓰고, Pretendard에 없는 문자이기 때문에 지원하지 않으며, 예정된 계획이 없습니다.
한자 같은 경우 지금은 간체, 번체, 홍콩 문자를 지원하지 않습니다. 추후 지원 예정이긴 하나, 진행하고 있는 일이 많아 언제쯤 시작할 수 있을지 불투명한 상황입니다.

질문에 도움이 되었으면 좋겠습니다. 추가적으로 궁금한 게 있으시면 언제든지 알려주세요!

또한 이 질문은 Discussion에서 이야기하는 것이 자연스러울 듯해 Discussion으로 옮겨두었습니다.
